Pedersen_N02/AmrWind/abl.inp

```diff
@@ -77,3 +77,81 @@ diffusion.mg_rtol = 1.0e-6
 diffusion.mg_atol = 1.0e-12
 temperature_diffusion.mg_rtol = 1.0e-10
 temperature_diffusion.mg_atol = 1.0e-13
+
+# Activate data probe sampling
+incflo.post_processing = sampling
+
+# Frequency of output for the data
+sampling.output_frequency = 5
+# Type of probes to output
+sampling.labels = l_v11 l_v12 l_v13 l_v21 l_v22 l_v23 l_v31 l_v32 l_v33 p_h p_v1 p_v2
+
+# Fields to output
+sampling.fields = velocity temperature
+
+# Definitions for each probe
+sampling/l_v11.type = LineSampler
+sampling/l_v11.num_points = 192
```

Comment:
```suggestion
sampling/l_v11.num_points = 201
```

Reply:
We want the same number of points as the vertical resolution I believe. For e.g. https://github.com/Exawind/wind-energy/blob/master/Pedersen_N02/NaluWindRun01/NaluWind_N02_velocity.dat has only 193 rows.

Reply:
@lawrenceccheung Could you confirm this. I believe you're sampling at all the nodes in Nalu-wind. Hence the 193 rows right? So amr-wind should have only 192?

Reply:
@gantech Oh, I should clarify, the output of the horizontally averaged files (like [NaluWind_N02_velocity.dat](https://github.com/Exawind/wind-energy/blob/master/Pedersen_N02/NaluWindRun01/NaluWind_N02_velocity.dat)) ultimately comes from the `boundary_layer_statistics` algorithm in Nalu-Wind, and not from the `data_probes` we put in for sampling spectra.  

For the sampling of spectra in `data_probes`, Alan and I  _oversampled_ the domain on purpose (which is why we put 201 points for 192 cells in height).  That way we still have at least one sample per cell, we avoid any possible issues with data points hitting the exact cell boundaries, and the heights come out in nice, round 10m increments.